app/views/tag/_miniCard.html.erb

```diff
@@ -46,6 +46,11 @@
     <% elsif !tag.name.include? ':' %>
       <a class="pull-right btn btn-primary requireLogin" href='/subscribe/tag/<%= tag.name %>'><%= translation('tag.show.follow') %></a>
     <% end %>
+    <% if tag.name == "one" %>
```

Comment:
Ah, here, could we make it completely parametric, that is, could we say, if it matches ANY tag listed in the LINKS file, could we display it? It might be something like:

```ruby
if LINKS.keys.include?(tag.name)
```

I'm not 100% sure that would work, but something like it? Also, this is in the mini cards, i haven't seen that yet... would you mind adding a screenshot? Thank you! I thought the previous location had been in the upper right corner next to the wiki edit button, but this sounds interesting too!

Reply:
> ```ruby
> if LINKS.keys.include?(tag.name)
> ```


@jywarren 
This code did not work, but I figured this out by first checking if such a key exists and if so, checking if the sub-key exists using this:

```ruby
if LINKS.has_key?("tags") && LINKS["tags"].has_key?(tag.name)
```

It works now and looks like this - 

![image](https://user-images.githubusercontent.com/78212650/188283947-089252a3-3c7b-4107-8430-fa673fc0a52b.png)